api/foxpass_create_users.py

```diff
@@ -34,8 +43,18 @@ def main():
                 continue
             data = {'email': email, 'username': username}
             r = requests.post(URL + ENDPOINT, headers=header, data=json.dumps(data))
-            print '{}: {}'.format(email, r.json())
+            try:
+                password = row[2]
+            except IndexError:
+                print '{}: {}'.format(email, r.json()['status'])
+                continue
+            if r.json()['status'] == 'ok' and password:
+                data = {'password': password}
+                r = requests.put(URL + ENDPOINT + username + '/', headers=header, data=json.dumps(data))
+                print '{}: password set'.format(email)
+            else:
+                print '{}: {}'.format(email, r.json()['status'])
```

Comment:
This line shouldn't be repeated if we can help it//condense it. I would set password = None before line 46, then if there's an IndexError just 'pass'. That way we only do a status & password check once.

Also add some comments as to what the if/else statements on lines 51 & 56 are doing.